aldryn_events/cms_plugins.py

```diff
@@ -27,7 +35,31 @@
     'as an apphook for that config.')
 
 
-class UpcomingPlugin(CMSPluginBase):
+class NameSpaceCheckMixin(object):
+
+    def get_namespace(self, instance):
+        namespace = instance.app_config_id and instance.app_config.namespace
```

Comment:
Not the most easy to read code. Also it can return False, where it is expected to return None.